cmd/snap-bootstrap/partition/encrypt.go

```diff
@@ -77,7 +99,7 @@ func NewEncryptedDevice(part *DeviceStructure, key EncryptionKey, name string) (
 		Node: fmt.Sprintf("/dev/mapper/%s", name),
 	}
 
-	if err := cryptsetupFormat(key, part.Node); err != nil {
+	if err := cryptsetupFormat(key, name+"-enc", part.Node); err != nil {
```

Comment:
Where does `-enc` come from? Is that our invention or is that something the system provides?

Reply:
@xnox named it. It's there to make it different from the unencrypted label because `/dev/disk/by-label/ubuntu-data` will also exist once the encrypted device is opened.

Reply:
More importantly, it should be a standard suffix for an encrypted foo. It could be anything, I picked `-enc` as that is often used with crypsetup / dmsetup names, but we can change it to something else. It does need to be a suffix, cause e.g. we will have potentially in the future new encrypted roles which will need a label pair for the LUKS & the filesystem inside the LUKS.